### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: iina/MPVNode.swift [hunk 1]
```diff
@@ -224,12 +224,12 @@ extension mpv_node: CustomDebugStringConvertible {
 
   /// A textual representation of this instance, suitable for debugging.
   ///
-  /// If the node is an array or a map the string will contine mulitple lines.
+  /// If the node is an array or a map the string will continue multiple lines.
```
Comment (low-batt) [inline iina/MPVNode.swift]: Oh no! This typu was added by low-batt!! It is such a bad typo that it looks like continue was misspelled, but it should be "contain", not continue.

File: iina/MPVNode.swift [hunk 2]
```diff
@@ -224,12 +224,12 @@ extension mpv_node: CustomDebugStringConvertible {
...
   public var debugDescription: String { mpv_node.toString(self) }
 
   /// Return a textual representation of the given`mpv_node`, suitable for debugging.
   ///
-  /// If the node is an array or a map the string will contine mulitple lines. For arrays and maps the string will include all nodes
+  /// If the node is an array or a map the string will continue multiple lines. For arrays and maps the string will include all nodes
```
Comment (low-batt) [inline iina/MPVNode.swift]: Same as above. The intent was to indicate "will contain multiple lines".

**Review by low-batt — APPROVED** [review]
Changes look good. Could change "continue" to "contain" as noted.
Thanks for catching these typos!

**Comment by low-batt** [comment]
@rex4539 Thanks for the update!

**Comment by rex4539** [comment]
@lhc70000 want to take a look?

**Pull request closed without merging**